static/app/utils/discover/fields.tsx

```diff
@@ -1103,7 +1103,7 @@ export function aggregateFunctionOutputType(
     return STARFISH_FIELDS[firstArg].outputType;
   }
 
-  if (!firstArg && STARFISH_AGGREGATION_FIELDS[funcName]) {
+  if (STARFISH_AGGREGATION_FIELDS[funcName]) {
```

Comment:
@DominikB2014 do you remember why the `!firstArg` condition is there? I couldn't think of a reason why a known Starfish aggregation field would need to also supply a `firstArg` to be looked up

Reply:
Tbh I can't recall a reason to have it, might have been something copy and pasted.